src/win/fs.c

```diff
@@ -2433,43 +2424,52 @@ static void fs__chmod(uv_fs_t* req) {
   }
 
   /* Create an ACE for each triplet (user, group, other) */
-  numNewEAs = 8 + 3*numOtherGroups;
+  numNewEAs = 7 + 3*numOtherGroups;
   ea = (PEXPLICIT_ACCESS_W) malloc(sizeof(EXPLICIT_ACCESS_W)*numNewEAs);
   u_mode = ((req->fs.info.mode & S_IRWXU) >> 6);
   g_mode = ((req->fs.info.mode & S_IRWXG) >> 3);
-  o_mode = ((req->fs.info.mode & S_IRWXO) >> 0);
+
+  /*
+   * Because we do not control the ordering of ACE entries within the ACL that
+   * we're building, the `SetNamedSecurityInfoW()` function call below will
+   * place all DENY entries first, and all `ALLOW` entries second.  This
```

Comment:
I think you mean `SetEntriesInAcl`, which we should perhaps just avoid due to this (like cygwin)

Reply:
From my initial explorations, if we don't use `SetEntriesInAcl`, it's a _lot_ more work.  We have to manually merge entries and such that I would much prefer not to do.  The Cygwin function to do this is many hundreds of lines of code; not counting their ACL datatype abstraction.  I don't actually care about this case (others with more permissions than group) very much, so I'm alright with just not supporting it and doing something less-broken than before.